myhoard/controller.py

```diff
@@ -1152,7 +1178,11 @@ def _mark_periodic_backup_requested_if_interval_exceeded(self):
         half_backup_interval_s = self.backup_settings["backup_interval_minutes"] * 60 / 2
         if (
             last_normalized_backup_time != normalized_backup_time
-            and self.state["backup_request"].get("normalized_backup_time") != normalized_backup_time
+            and (
+                not self.state["backup_request"]
+                or self.state["backup_request"]["normalized_backup_time"]  # pylint: disable=unsubscriptable-object
```

Comment:
Here too, why mypy can't see that "backup_request" has the type we expect ?

Reply:
mypy has no problems with it, but pylint does for some reason - same as above.